config/routes.rb

```diff
@@ -158,11 +161,13 @@
         patch :approve_channel
         get :statement
         post :create_note
+        get :cancel_two_factor_authentication_removal
```

Comment:
Really think these should just be routes on a `resources :two_factor_authentication_removal` controller

Reply:
You are right, I will revisit this in a later PR and make this refactor